### ISSUE TITLE
Ctivo does not show Tivo Bolt recorded shows

### ISSUE THREAD

**Issue by TivoOrlo** [dscottbuch/cTiVo#305]
Just upgraded to Ctivo 3.1.0 and I have a Tivo Boly, and none of my recorded shows on my Tivo Bolt show up in the Ctivo window. I've manually put in the IP Address in the Ctivo Preferences, but still no luck.

Any suggestions?

Thanks.

**Comment by mackworth** [comment]
Does the Bolt itself show up, or do you get a message about “can’t find TiVo”? Do you have just the one? Can you set the debug level to Detail, quit and start again? ( I would delete the manual one first) then post the log, so I can see what’s happening.

**Comment by TivoOrlo** [comment]
Hi, Hugh,

Thanks for getting back to me.

The Tivo Bolt does show up, and I have only one.

I followed your instructions and am attaching the Logs.

Thanks,

Chad Orlowski

**Comment by mackworth** [comment]
Unfortunately, Github strips attachments on emails; you have to post them at the site.

**Comment by TivoOrlo** [comment]
Here are the logs.
[com.cTiVo.cTiVo 2018-02-11 21-11.log](https://github.com/dscottbuch/cTiVo/files/1717310/com.cTiVo.cTiVo.2018-02-11.21-11.log)

**Comment by mackworth** [comment]
So, it looks like it was having a problem prior to 3.1.0 update. It looks like the known anti-virus problem (AV programs won't allow access to self-certified SSL ports like TiVo).
If you are running AV, then you need to white-list your TiVo.

Try this link in your browser:  [Your TiVo](https://dvr-9327.local./TiVoConnect?Command=QueryContainer&Container=%252FNowPlaying&Recurse=Yes&AnchorOffset=0&ItemCount=15)
It may work, or it may give you a warning.

**Comment by TivoOrlo** [comment]
That was the problem!
However, I am having trouble whitelisting Tivo onKaspersky. In the meantime, I shut Kaspersky off temporarily, and got the following incomplete list of programs. I don't know what the yellow ! means.
![ctivco screenshot](https://user-images.githubusercontent.com/36408578/36117597-3ef99292-1008-11e8-970b-adce3f900576.png)

**Comment by mackworth** [comment]
Glad that was the cause. When you say”incomplete “, what’s missing?

The exclamation mark is just your TiVo saying that show it might be deleted, same as on your NowPlaying screen.

**Comment by TivoOrlo** [comment]
I say "incomplete," because I cannot do anything to the programs listed in the "top" CTivo window (Now Playing list), and I cannot proceed with the download to my computer in the "bottom" window (Download Queue) (Notice the "Failed" listing in the DL Stage).

I'm trying different things but cannot get CTivo to work completely.

**Comment by mackworth** [comment]
So to help on the Failed, I’ll need a log showing the failure. Might try Default as a Format rather than “DVD-ready”

On the top window, it’s mostly intended as information and as a source from which you can create Downloads or Subscriptions

**Comment by TivoOrlo** [comment]
Here is the log trying to Download the program Super Bowl LII which failed.

Thanks.
[com.cTiVo.cTiVo 2018-02-12 22-47.log](https://github.com/dscottbuch/cTiVo/files/1718137/com.cTiVo.cTiVo.2018-02-12.22-47.log)

**Comment by mackworth** [comment]
Well, that was easy!

> 2018-02-12 19:10:56:646 MTTiVoManager>notifyForName:withTitle:subTitle:isSticky:@1401>Notify: Pausing downloads: Your temporary or boot drive is low on space
> Probably need to delete some files.
> 2018-02-12 19:10:56:647 MTDownload>configureFiles@625>Disk space problem: 4,633,722,880 tmp and 108,832,235,520 download vs 29,999,759,360 fileSize
> 
So, that's saying you're trying to download a 30G file with only 4G left on your temporary (usually your boot) drive. cTiVo needs space to download the file before then encoding it.  You can move the temporary directory in advanced Preferences, but you might want to clear up your boot drive anyways; 4G is a little small nowadays!


You should have gotten a warning in your Notifications (Upper right menu bar). Is it not there?

**Comment by TivoOrlo** [comment]
Well, Hugh,

I give you kudos and olive branches for successfully navigating me thru this difficult morass of technology. You have hit the nail on the head, and the program is successfully downloading.

I have used your program in the past and thought it quite useful, but this is the first time I've encountered difficulties, but your support is quite extraordinary in today's world. 

Let me know if I can help you in any way in your endeavors. I am not that technologically astute but, be that as it may, my offer still holds.

Thanks,

Chad Orlowski
